front/app/modules/commercial/bulk_idea_import/admin/components/Tab.tsx

```diff
@@ -21,6 +21,7 @@ const Tab: FC<Props & WrappedComponentProps> = ({
           name: 'import',
           label: formatMessage(messages.import),
           url: '/admin/ideas/import',
+          statusLabel: 'Beta',
```

Comment:
Minor: wondering if Beta is universally used in other languages as well okay as it is too

Reply:
Oops, added it here by mistake. In terms of texting, it's technically supported only in the US, so probably it's fine to not translate it.